### ISSUE TITLE
Create on chain validation for avro schema via no_std version

### ISSUE THREAD

**Issue by saraswatpuneet** [LibertyDSNP/mrc#107]
Tasks
- [ ] Register schema to do avro validation using no_std version of rust avro library 
- [ ] Update rpc endpoint to also utilize on chain schema validation

Dependent on #67

**Comment by saraswatpuneet** [comment]
Duplicate of the following issue, so closing this one
#29
